docs/internal/release.rst

```diff
@@ -208,8 +210,10 @@ Before Running the Release Script
     - Git version 2.5 or later
     - :ref:`Python 3 for News Generation <docs-requirements>`
 
-  - Your GitHub account has been added as a member of the `OpenDDS organization <https://github.com/OpenDDS>`__ with the appropriate permissions.
+  - Your GitHub account meets the following requirements:
 
+    - It has been added as a member of the `OpenDDS organization <https://github.com/OpenDDS>`__ with the appropriate permissions.
+    - It has permissions to update the release artifacts for the `OMG RTPS Interop repo <https://github.com/omg-dds/dds-rtps>`__ for :ref:`release-upload-artifacts`.
```

Comment:
If the release script command-line arguments indicate that the user want to try uploading, but the user doesn't actually have permissions, does the release continue with a warning or error out?

Reply:
This is happening after the release proper and it will error when it tries to upload to GitHub.